COLLABORATOR_GUIDE.md

```diff
@@ -831,7 +831,7 @@ LTS working group and the Release team.
 | Subsystem                                | Maintainers                                                           |
 | ---                                      | ---                                                                   |
 | `benchmark/*`                            | @nodejs/benchmarking, @mscdex                                         |
-| `bootstrap_node.js`                      | @nodejs/process                                                       |
+| `bootstrap/node.js`                      | @nodejs/process                                                       |
```

Comment:
Maybe just: `bootstrap/*`?

Reply:
I am not sure: previous `bootstrap_loaders.js` was not mentioned here. Is it `@nodejs/modules` domain, not ` @nodejs/process`?
Refs: 8484b40b3d38e18c524b7dd560c16ab30c94e427

Reply:
The loaders were originally part of `bootstra_node.js`

Reply:
However, it seems your suggestion is valis as per https://github.com/nodejs/node/pull/20554
So I will replace with `bootstrap/*`.

Reply:
Amended.